RealtimeJMSToStream/README.rst

```diff
@@ -0,0 +1,85 @@
+Realtime Java Message Service (JMS) Source To Stream Sink Adapter Configuration
```

Comment:
Add a line of equal signs above the title. (See the other READMEs in this repo.)